common/models/user.js

```diff
@@ -24,6 +24,12 @@ class User {
     return this
   }
 
+  setArea({ areaCode, areaName } = {}) {
+    this.areaCode = areaCode
+    this.areaName = areaName
+    return this
+  }
+
   getDetails() {
```

Comment:
Are we including the `areaCode` and `areaName` on this?

Reply:
should we separate user details / user auth context all together? I think that will be a cleaner approach. what do you think?

Reply:
It's nice to encapsulate them them in a single user object on the request, but agree it could be cleaner - happy to catch up on this